packages/core/src/plugins/algolia.js

```diff
@@ -24,11 +24,12 @@ function addNoIndexClasses(content) {
     'dropdown',
     'b-modal',
     'panel:not([expanded])',
-    'popover div[slot=content]',
+    // to target popover and tooltips
+    '[data-mb-component-type] [data-mb-slot-name="content"], [data-mb-slot-name="_content"]',
```

Comment:
how about:

```suggestion
    '[data-mb-component-type] [data-mb-slot-name]',
```
so the popover `header`s don't get indexed too

Reply:
Thanks for the review! Made the changes as per our discussion :-)